src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -18,18 +33,97 @@ namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
     internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
     {
         public const string TagId = "inline parameter name hints";
+        private readonly IToolTipService _toolTipService;
+        private readonly ITextView _textView;
+        private SnapshotSpan _span;
+        private readonly SymbolKey _key;
+        private readonly IThreadingContext _threadingContext;
+        public Lazy<IStreamingFindUsagesPresenter> _streamingPresenter;
 
         /// <summary>
         /// Creates the UIElement on call
```

Comment:
Comment is now stale? It might be best to just move it to the Create method -- usually if there's a public create it's OK to then leave the constructor uncommented because that's just not as interesting anymore.